server/interceptor/interceptor.go

```diff
@@ -265,6 +265,38 @@ type rpcInfo struct {
 	tokens int
 }
 
+// chargable is satisfied by request proto messages which contain a GetCharge
+// accessor.
+type chargable interface {
+	GetCharge() *trillian.Charge
+}
+
+// addChargedUsers adds quota spec entries for any chargable user quotas.
```

Comment:
When the dust settles on interceptor.go it will probably make sense to have a single function that does all of the quota-spec building.

Reply:
ack